api/route/pokemons.route.ts

```diff
@@ -0,0 +1,96 @@
+import { Router } from "express";
+import { getRandomPokemon } from "../controller/pokemons.controller";
+import { validateUserId } from "../validation/pokemons.validation";
```

Comment:
import pokemonsValidation and then pokemonsValidation.validateUserId

the validation functions name should be on the route generally and not coupled to the userId,
tomorrow we could add another parameter and then the name validateUserId will not match